<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1597</width>
    <height>936</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="joueur_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Cahier_de_jeu_rugby_Noah/img/1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>0</y>
      <width>431</width>
      <height>761</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>429</width>
       <height>759</height>
      </rect>
     </property>
     <widget class="QPushButton" name="start">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
     <widget class="QLabel" name="x">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Dernier Point = </string>
      </property>
     </widget>
     <widget class="QLabel" name="y">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Premier Point = </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Distance (joueur 2) :</string>
      </property>
     </widget>
     <widget class="QLabel" name="pos_souris">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>620</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Position souris : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="joueur1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>joueur 1 </string>
      </property>
     </widget>
     <widget class="QPushButton" name="joueur2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>joueur 2 </string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QLabel" name="distance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Distance (joueur 1) : </string>
      </property>
     </widget>
     <widget class="QLabel" name="distance_ballon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Distance (ballon) :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Courir">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Courir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Marcher">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Marcher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Trottiner">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Trottiner</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_joueur">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>reset_joueur</string>
      </property>
     </widget>
     <widget class="QLabel" name="alerte">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>680</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string> Alerte : impossibilit√© physique!!!</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="vitesse_ballon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Vitesse du ballon : </string>
      </property>
     </widget>
     <widget class="QLabel" name="vitesse_joueur1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Vitesse du joueur 1 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="vitesse_joueur2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Vitesse du joueur 2 :</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>20</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="joueur_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Cahier_de_jeu_rugby_Noah/img/14.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ballon">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Cahier_de_jeu_rugby_Noah/img/ballon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
